foreman/foreman.spec

```diff
@@ -190,8 +190,9 @@ BuildRequires: %{?scl_prefix_ror}rubygem(execjs) >= 1.4.0
 BuildRequires: %{?scl_prefix_ror}rubygem(execjs) < 2.5.0
 BuildRequires: %{?scl_prefix_ror}rubygem(jquery-rails) >= 3.1
 BuildRequires: %{?scl_prefix_ror}rubygem(jquery-rails) < 4.0
+BuildRequires: %{?scl_prefix_ror}rubygem(bootstrap-sass) >= 3.3.5
```

Comment:
This shouldn't be in foreman.spec should it?  It looks like there's no dep in Foreman, only in patternfly-sass.